homeassistant/components/blueprint/websocket_api.py

```diff
@@ -21,12 +24,15 @@ def async_setup(hass: HomeAssistant):
     """Set up the websocket API."""
     websocket_api.async_register_command(hass, ws_list_blueprints)
     websocket_api.async_register_command(hass, ws_import_blueprint)
+    websocket_api.async_register_command(hass, ws_save_blueprint)
+    websocket_api.async_register_command(hass, ws_delete_blueprint)
 
 
 @websocket_api.async_response
 @websocket_api.websocket_command(
     {
         vol.Required("type"): "blueprint/list",
+        vol.Optional("domain"): cv.string,
```

Comment:
Let's just change this to be required. 
```suggestion
        vol.Required("domain"): cv.string,
```

We can always add a catch all available blueprints when we need it.